src/layouts/default.hbs

```diff
@@ -24,4 +24,10 @@
 {{> main}}
 {{> footer}}
   </body>
+{{#if env.IS_DOCS}}
+  <!-- Google Tag Manager (noscript) -->
+  <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-NXSDZWN"
+  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+  <!-- End Google Tag Manager (noscript) -->
+{{/if}}
```

Comment:
Sorry if I was unclear. This part needs to be right after the **opening** body tag, not the closing one.

Reply:
Np.

Reply:
Done.